.github/workflows/build-ci.yml

```diff
@@ -59,6 +59,16 @@ jobs:
 
         steps:
             -   uses: actions/checkout@v2
+            -   name: Creating MongoDB replica
+                run: |
+                    docker run --name mongodb_repl -e MONGO_INITDB_DATABASE=unittest --publish 27018:27018 --detach mongo:${{ matrix.mongodb }} mongod --port 27018 --replSet rs
```

Comment:
OK, then I think this should definitely be changed to avoid additional overhead (IMO it's a waste of CI resources) and simplify some of the logic in the test files.

Reply:
I have already setup it in `docker-compose.yml` for local testing and it works. But in `build-ci.yml` we have some problem, I am not understand problem at all yet but it related with access to mongodb container in run tests step. I need more time to make it work properly